src/pages/overview/Overview.jsx

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react';
+import posts from '../../constants/data.json';
+import {Link} from "react-router-dom";
+import axios from "axios";
+
+function Overview() {
+    const [posts, setPosts] = useState([])
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState('')
+
+        async function fetchPosts() {
+            setLoading(true);
+            setError('');
+            try {
+                const response = await axios.get ('http://localhost:3000/posts');
+                setPosts(response.data);
+                console.log(response.data);
+            } catch (e) {
+                console.log(e)
+                setError('Something went wrong: ' + e.message);
+            } finally {
+                setLoading(false);
+            }
+        }
+
+    useEffect(() => {
+    fetchPosts();
+    }, []);
+
+
+    return (
+        <section className={'overview-section outer-content-container'}>
+            <div className={'inner-content-container'}>
+                <button type={'button'} onClick={fetchPosts}>Get all posts</button>
```

Comment:
Is deze wel nodig? De functie wordt namelijk al direct aangeroepen en uitgevoerd zodra de pagina wordt geladen vanwege de useEffect op regel 26.